modules/tinymce/src/plugins/help/main/ts/ui/PluginsTab.ts

```diff
@@ -9,8 +9,8 @@ import * as PluginUrls from '../data/PluginUrls';
 
 const tab = (editor: Editor): Dialog.TabSpec & { name: string } => {
   const availablePlugins = () => {
-    const premiumPlugins = Arr.filter(PluginUrls.urls, ({ key, type }) => {
-      return key !== 'autocorrect' && type === PluginUrls.PluginType.Premium;
```

Comment:
Why was this in there in the first place, the previous plugin name has the correct type as well?

Reply:
It was there so it didn't show up in the Premium Plugin text based list so it didn't show, but still showed in the dynamic plugins when loaded for the specific customers that did have it.